<ui version="4.0" >
 <class>CheatDialog</class>
 <widget class="QDialog" name="CheatDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cheats</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <layout class="QGridLayout" name="gridLayout_2" >
     <item row="2" column="3" >
      <widget class="QPushButton" name="pushDelete" >
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" >
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5" >
      <widget class="QTextEdit" name="textDescription" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5" >
      <widget class="QTreeView" name="treeView" >
       <property name="animated" >
        <bool>true</bool>
       </property>
       <property name="headerHidden" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QPushButton" name="pushNew" >
       <property name="text" >
        <string>&amp;New</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="pushEdit" >
       <property name="text" >
        <string>&amp;Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CheatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CheatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
